java/client/test/org/openqa/selenium/PrintPageTest.java

```diff
@@ -0,0 +1,71 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assume.assumeTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.print.PageSize;
+import org.openqa.selenium.print.PrintOptions;
+import org.openqa.selenium.testing.JUnit4TestBase;
+
+public class PrintPageTest extends JUnit4TestBase {
+  private static String MAGIC_STRING = "JVBER";
+  private PrintsPage printer;
+
+  @Before
+  public void setUp() {
+    assumeTrue(driver instanceof PrintsPage);
+    printer = (PrintsPage) driver;
+    driver.get(pages.printPage);
+  }
+
+  @Test
+  public void canPrintPage() {
+    PrintOptions printOptions = new PrintOptions();
+
+    Pdf pdf = printer.print(printOptions);
+    assertThat(pdf.getContent().contains(MAGIC_STRING)).isTrue();
+  }
+
+  @Test
+  public void canPrintwoPages() {
+    PrintOptions printOptions = new PrintOptions();
+    printOptions.setPageRanges(new String[]{"1-2"});
```

Comment:
If this is how we're expecting people to use this, then consider making `setPageRanges` take a mandatory first argument and String varargs to allow something like `setPageRanges("1-2", "5-8")`